### ISSUE TITLE
CPU usage

### ISSUE THREAD

**Issue by raziel29a** [drgg/noita-LCAP#1]
30% CPU usage is absolutely unacceptable with a program that is supposed to be used once per run. What is this? A bitcoin miner?

**Comment by drgg** [comment]
No, it is not, you can see the code publicly and compile it yourself if you please. But, I was not aware of the high CPU usage, I will look in to it in a future release.

**Comment by raziel29a** [comment]
I wasn't entirely serious about the bitcoin miner :) can't do that on GitHub. I hope it gets fixed. The first while is an endless loop until F4 is pressed. It's just running as fast as the CPU can and do nothing. Rewrite the code so it waits for input before going any further or if you're lazy just drop a 1000ms wait in that loop.

**Comment by drgg** [comment]
Okay! Thank you, I will do exactly that
